sdk/docs/pages/typescript/cryptography/keypairs.mdx

```diff
@@ -125,3 +125,49 @@ const exampleMnemonic = 'result crisp session latin ...';
 
 const keyPair = Ed25519Keypair.deriveKeypair(exampleMnemonic);
 ```
+
+## Exporting and importing key pairs
```

Comment:
So I know this technically is supported, but I often wonder if this really should be a method, or if we should just let you get the private key bytes and do this work yourself. We added this mostly for the wallet, but the more I see it the less I'm convinced it should be built in.